drill/validate.sh

```diff
@@ -82,5 +90,5 @@ if ((ui_result == 200 && sql_result == 0)); then
   exit 0
 fi
 
-echo "Test failed"
+echo "  failed"
```

Comment:
reason you want to remove the word "Test" and put two empty spaces?

Reply:
By mistake, reverted.